pages/gallery.html

```diff
@@ -4,9 +4,76 @@
   <meta http-equiv="X-UA-Compatible" content="IE=edge">
   <meta name="viewport" content="width=device-width, initial-scale=1.0">
-  <title>Document</title>
+  <title>Pics of MILFs</title>
+  <link rel="stylesheet" href="../styles/styles.css">
+  <link rel="stylesheet" href="../styles/gallery.css">
 </head>
 <body>
-  
+  <div id="wrapper">
+    <div class="container">
+      <img src="../assets/mushroom.png" class="shroom" style="--i:21;"></img>
+      <img src="../assets/mushroom.png" class="shroom" style="--i:12;"></img>
+      <img src="../assets/mushroom.png" class="shroom" style="--i:18;"></img>
+      <img src="../assets/mushroom.png" class="shroom" style="--i:10;"></img>
+      <img src="../assets/mushroom.png" class="shroom" style="--i:14;"></img>
+      <img src="../assets/mushroom.png" class="shroom" style="--i:17;"></img>
+      <img src="../assets/mushroom.png" class="shroom" style="--i:18;"></img>
+      <img src="../assets/mushroom.png" class="shroom" style="--i:22;"></img>
+      <img src="../assets/mushroom.png" class="shroom" style="--i:18;"></img>
+      <img src="../assets/mushroom.png" class="shroom" style="--i:15;"></img>
+      <img src="../assets/mushroom.png" class="shroom" style="--i:14;"></img>
+      <img src="../assets/mushroom.png" class="shroom" style="--i:25;"></img>
+      <img src="../assets/mushroom.png" class="shroom" style="--i:13;"></img>
+      <img src="../assets/mushroom.png" class="shroom" style="--i:21;"></img>
+      <img src="../assets/mushroom.png" class="shroom" style="--i:25;"></img>
+      <img src="../assets/mushroom.png" class="shroom" style="--i:16;"></img>
+      <img src="../assets/mushroom.png" class="shroom" style="--i:15;"></img>
+      <img src="../assets/mushroom.png" class="shroom" style="--i:12;"></img>
+      <img src="../assets/mushroom.png" class="shroom" style="--i:19;"></img>
+      <img src="../assets/mushroom.png" class="shroom" style="--i:20;"></img>
+      <img src="../assets/mushroom.png" class="shroom" style="--i:5;"></img>
+      <img src="../assets/mushroom.png" class="shroom" style="--i:13;"></img>
+      <img src="../assets/mushroom.png" class="shroom" style="--i:20;"></img>
+      <img src="../assets/mushroom.png" class="shroom" style="--i:7;"></img>
+      <img src="../assets/mushroom.png" class="shroom" style="--i:16;"></img>
+    </div>
+
+    <header>
+      <nav>
+          <a href="index.html">Home Page</a></li>
+          <a href="facts.html">Fun Facts</a></li>
+      </nav>
+      <h1>Hot MILFs In Your Area!</h1>
+      <div class="gif-container">
+        <img class="gif" src="../assets/giphy.gif">
+        <img class="gif" src="../assets/giphy.gif">
+        <img class="gif" src="../assets/giphy.gif">
+      </div>
+    </header>
+
+    <main>
+      <h2 class="item1">Edible</h2>
+      <div class="item2">
+        <img class="mushys" src="../assets/Magic_Mushroom_Collection-732x549-Thumbnail.webp">
+        <figcaption>yummy mushrooms</figcaption>
```

Comment:
If using a `figcaption`, the image and the caption should be inside a `<figure>` tag rather than a `<div>`. See more here: https://developer.mozilla.org/en-US/docs/Web/HTML/Element/figure